src/components/events/UpcomingEvents/index.js

```diff
@@ -93,10 +101,7 @@ module.exports = React.createClass({
 
   onEventLoad: function(result) {
     if (!result.campaigns) { return; }
-    var sorted = _.sortBy(_.filter(result.campaigns, this.filterEvents), function(e) {
-      return new Date(e.display_start_at);
-    });
-    this.setState({ events: sorted });
+    this.setState({ events: this.sortEvents(this.filterEvents(result.campaigns)) });
```

Comment:
Nice!

Side note (idle curiosity), this is a good candidate for `compose` :D

```
function compose (fnA, fnB, thisArg) {
  return function (a) { return fnA.call(thisArg, fnB.call(thisArg, a)) }
}
this.filterAndSortEvents: compose(this.filterEvents, this.sortEvents, this)
```

:D